docs/user-guide/README.md

```diff
@@ -64,6 +64,9 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
 [**Overview**](overview.md)
 : A brief overview of Kubernetes concepts.
 
+**Cluster**
+: A cluster is the compute resources on top of which your containers are built.
```

Comment:
This definition of cluster is from [kubernetes/README.md](https://github.com/GoogleCloudPlatform/kubernetes#concepts). I'll modify the definition of Cluster and add Node definition in /README.md too.